fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1311,4 +1311,9 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static boolean enable_fe_heartbeat_by_thrift = false;
+
+    /**
+     * default value for insert_visible_timeout_ms
+     */
+    public static long default_insert_visible_timeout_ms = 10000;
```

Comment:
If we use session variable, there is no need to add this config.
I think session variable is enough.